conf/streamline-dev.yaml

```diff
@@ -120,6 +126,7 @@ logging:
   loggers:
     # Sets the level for 'com.example.app' to DEBUG.
     com.hortonworks.streamline: DEBUG
+    org.apache.tomcat.jdbc.pool.PooledConnection: ERROR
```

Comment:
flyway bundle asks for a DataSource instance where we have to mention the driver class, if the driver class is not mentioned,  driver class is inferred from URL of the database but it throws warnings to the console that the driver class is null, which could be misleading the user, so I have suppressed the warnings from PooledConnection.